docs/topics/Permissions.md

```diff
@@ -59,6 +59,7 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | MANAGE_ROLES \*       | `0x10000000` | Allows management and editing of roles                                                                                             | T, V         |
 | MANAGE_WEBHOOKS \*    | `0x20000000` | Allows management and editing of webhooks                                                                                          | T, V         |
 | MANAGE_EMOJIS \*      | `0x40000000` | Allows management and editing of emojis                                                                                            |              |
+| USE_SLASH_COMMANDS    | `0x80000000` | Allows members to use slash commands in text channels.                                                                             | T            |
```

Comment:
Remove the `.` to be consistent with the rest of the table?

Reply:
```suggestion
| USE_SLASH_COMMANDS    | `0x80000000` | Allows members to use slash commands in text channels                                                                              | T            |
```